src/Raven.Client/Documents/Session/DocumentSession.IncludeCounters.cs

```diff
@@ -0,0 +1,46 @@
+//-----------------------------------------------------------------------
+// <copyright file="DocumentSession.cs" company="Hibernating Rhinos LTD">
+//     Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Raven.Client.Documents.Session.Loaders;
+
+namespace Raven.Client.Documents.Session
+{
+    /// <summary>
+    /// Implements Unit of Work for accessing the RavenDB server
+    /// </summary>
+    public partial class DocumentSession
+    {
+        public ILoaderWithInclude<object> IncludeCounter(string name)
```

Comment:
I do not think that we want those overloads without generic `T` here. Let's drop them.

Reply:
done